src/server/connectors/config-db/config-db.connector.js

```diff
@@ -110,6 +112,59 @@ const getPathConfigByVersion = async version => {
   return pathConfig;
 };
 
+/**
+ * Fetches the list of allowed local councils from the config database
+ *
+ * @returns {Array} An array containing shortened local council names
+ */
+const getLocalCouncils = async () => {
+  let localCouncilUrls;
+  logEmitter.emit("functionCall", "config-db.connector", "getLocalCouncils");
+
+  try {
+    await establishConnectionToMongo();
+
+    localCouncilUrls = await lcConfigCollection.distinct("local_council_url");
```

Comment:
This is because there are certain coucils that only exist in order to receive notifications (Dorset in this case) but won't be used to directly register with...part of the difference between a hygeine and standards council. I've got a spreadsheet on this I can send over.